audioapp/tests/test_views.py

```diff
@@ -1,85 +1,81 @@
                 'title': 'New Test Audio',
-                'description': 'Description',
-                'file': SimpleUploadedFile(fp.name, fp.read(), content_type='audio/mp4')
+                'description': 'New Description',
+                'file': file
             })
+        if response.status_code == 200:
+            print(response.context['form'].errors)
         self.assertEqual(response.status_code, 302)
         self.assertTrue(AudioFile.objects.filter(title='New Test Audio').exists())
     
     def test_audio_detail_view(self):
         response = self.client.get(reverse('audio_detail', args=[self.audio_file.pk]))
         self.assertEqual(response.status_code, 200)
         self.assertTemplateUsed(response, 'audio_detail.html')
-        self.assertContains(response, 'Test Audio')
+        self.assertContains(response, self.audio_file.title)
     
     def test_user_detail_view(self):
         response = self.client.get(reverse('user_detail', args=[self.user.username]))
         self.assertEqual(response.status_code, 200)
         self.assertTemplateUsed(response, 'user_detail.html')
-        self.assertContains(response, 'Test Audio')
-
-    #Like tests
+        self.assertContains(response, self.audio_file.title)
+    
     def test_like_audio(self):
-        self.client.login(username='testuser', password='testpassword')
         response = self.client.post(reverse('like_audio', args=[self.audio_file.pk]))
         self.assertEqual(response.status_code, 302)
         self.assertTrue(Like.objects.filter(user=self.user, audio_file=self.audio_file).exists())
     
     def test_unlike_audio(self):
-        # First, like the audio file
-        Like.objects.create(user=self.user, audio_file=self.audio_file)
-        self.client.login(username='testuser', password='testpassword')
+        like = LikeFactory(user=self.user, audio_file=self.audio_file)
         response = self.client.post(reverse('unlike_audio', args=[self.audio_file.pk]))
         self.assertEqual(response.status_code, 302)
         self.assertFalse(Like.objects.filter(user=self.user, audio_file=self.audio_file).exists())
     
-    def test_like_model(self):
-        like = Like.objects.create(user=self.user, audio_file=self.audio_file)
-        self.assertEqual(like.user, self.user)
-        self.assertEqual(like.audio_file, self.audio_file)
+    def test_create_comment(self):
+        comment = CommentFactory(user=self.user, audio_file=self.audio_file, text='This is a test comment.')
+        self.assertEqual(comment.user, self.user)
+        self.assertEqual(comment.audio_file, self.audio_file)
+        self.assertEqual(comment.text, 'This is a test comment.')
+
+    def test_comment_relationships(self):
+        comment = CommentFactory(user=self.user, audio_file=self.audio_file)
+        self.assertIn(comment, self.audio_file.comments.all())
+        self.assertIn(comment, Comment.objects.filter(user=self.user))
+    
```

Comment:
This file looks like it wants another new line at the end. Don't worry about it for this PR, but maybe set up some sort of Python formatter that could handle this for future work.

https://github.com/psf/black